aws-alb-ingress-controller/Makefile

```diff
@@ -1,9 +1,23 @@
+KUBERNETES_VERSION="1.18.1"
```

Comment:
The schema version used by kubeval is only `1.18.1`
https://github.com/instrumenta/kubernetes-json-schema